base/div.jl

```diff
@@ -67,6 +67,7 @@ rem(x, y, r::RoundingMode)
 rem(x, y, ::RoundingMode{:ToZero}) = rem(x,y)
 rem(x, y, ::RoundingMode{:Down}) = mod(x,y)
 rem(x, y, ::RoundingMode{:Up}) = mod(x,-y)
+rem(x, y, r::RoundingMode{:Nearest}) = x - y*round(x/y, r)
```

Comment:
This should probably be
```suggestion
rem(x, y, r::RoundingMode{:Nearest}) = x - y*div(x, y, r)
```
to avoid the intermediate rounding as claimed in the docstring?

Reply:
div(x, y, r) is giving me errors. Does div(x, y, r) exist?

Reply:
Yes, on current `master` (or `nightly`). I was running into the same issue, until I realized that there might have been additions recently.

Reply:
Oh yes, you are right. I thought I was running the latest build but I wasn't. Sorry!